src/uncrustify.cpp

```diff
@@ -479,6 +480,36 @@ int main(int argc, char *argv[])
       }
    }
 
+   /* Set config options using command line arguments.*/
+   idx = 0;
+   while ((p_arg = arg.Params("--set", idx)) != NULL)
+   {
+      char buffer[256];
+      strcpy(buffer, p_arg);
+
+      char *ptr;
+      if ((ptr = strchr(buffer, '=')) != NULL)
+      {
+         // Tokenize the string using the null character and move to second token.
+         *ptr++ = '\0';
+
+         if (*ptr == '\0')
+         {
+            usage_exit("Error while parsing --set", argv[0], 64);
```

Comment:
If you were wondering what's up with the `64` magic exit code.

I was trying to make sense of the current exit codes used in Uncrustify and I've found that they could be related to the FreeBSD preferable exit codes: http://www.freebsd.org/cgi/man.cgi?query=sysexits&apropos=0&sektion=0&manpath=FreeBSD+4.3-RELEASE&format=html

I think this should be refactored in a next PR even though it's not that important.